staging/traefik/values.yaml

```diff
@@ -17,7 +17,6 @@ testFramework:
 
 ## can switch the service type to NodePort if required
 serviceType: LoadBalancer
-extraServicePorts: []
```

Comment:
hum? KBA currently relies on this to specify the velero port 9000. Removing this without adjusting kba would break the user.